integrations/zendesk/src/actions/create-ticket.ts

```diff
@@ -0,0 +1,40 @@
+import { transformTicket } from 'src/definitions/schemas'
+import { getZendeskClient } from '../client'
+import { IntegrationProps } from '.botpress'
+
+export const createTicket: IntegrationProps['actions']['createTicket'] = async ({ ctx, client, input }) => {
+  const zendeskClient = getZendeskClient(ctx.configuration)
+  const ticket = await zendeskClient.createTicket(input.subject, input.comment, {
+    name: input.requesterName,
+    email: input.requesterEmail,
+  })
+
+  const { conversation } = await client.getOrCreateConversation({
+    channel: 'ticket',
+    tags: {
+      id: ticket.id.toString(),
+      originConversationId: input.__conversationId,
+    },
+  })
+
+  await zendeskClient.updateTicket(ticket.id, {
+    external_id: conversation.id,
+  })
+
+  const { user } = await client.getOrCreateUser({
```

Comment:
**chore:** I don't believe you have to create a user for the patient here, I think the bot should be your user